GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigAdvancedSettingsPage.cs

```diff
@@ -1,30 +1,54 @@
-﻿using Microsoft;
+﻿using System.Collections.Generic;
+using System.Windows.Forms;
+using Microsoft;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class GitConfigAdvancedSettingsPage : ConfigFileSettingsPage
     {
+        private record GitSettingUiMapping(string GitSettingKey, CheckBox MappedCheckbox);
+        private readonly List<GitSettingUiMapping> _gitSettings;
+
         public GitConfigAdvancedSettingsPage()
         {
             InitializeComponent();
             Text = "Advanced";
             InitializeComplete();
+
+            _gitSettings = new List<GitSettingUiMapping>
+            {
+                new("pull.rebase", checkBoxPullRebase),
+                new("fetch.prune", checkBoxFetchPrune),
+                new("rebase.autoStash", checkBoxRebaseAutostash),
+                new("rebase.autosquash", checkBoxRebaseAutosquash)
+            };
+            Load += GitConfigAdvancedSettingsPage_Load;
+        }
+
+        private void GitConfigAdvancedSettingsPage_Load(object? sender, System.EventArgs e)
+        {
+            foreach (GitSettingUiMapping gitSetting in _gitSettings)
+            {
+                gitSetting.MappedCheckbox.Text += $" [{gitSetting.GitSettingKey}]";
+            }
```

Comment:
This is Git config so different from GE settings.

Reply:
This piece of code is not to bind settings, it's just to update the checkbox label by passing the translation because to add some text (the git setting key) that should not be translated.

So it seems better for me for 2 reasons:
* we don't change the strings already translated (so no strings to translate again)
* string are easier to translate because the part that should not be translated is not in the string provided for translation.

And what @gerhardol say seems right also.